samples/04_gis_analysts_data_scientists/forecasting_air_temperature_in_california.ipynb

```diff
@@ -0,0 +1,2703 @@
+{
```

Comment:
<p>put these in one cell for Visualize as pandas dataframe</p><p>climate_data = data_table.tables[0]</p><p>climate_df = climate_data.query().sdf</p><p>climate_df.head()</p><p> </p><p>Also can you explain the dataset, defining the column names, what they represent</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/1119/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>